README.md

```diff
@@ -1 +1,20 @@
-# nlp-servic
\ No newline at end of file
+# NLP-Service for entity extraction using spacy or Polyglot depending on the language
+The Service provides entities and detects the language of text. Results are served by a flask app with two endpoints:
+`entity_extraction`, which returns the entities in the text, the used method/model and the language of the text. You can also add a language in the POST-request, in order to specify, which language should be used by the service.
+and
+`language_detection`, which runs polyglots language detection and returns it.
+
+### Running the service without Docker
+If you want, you can run the service locally, after creating a python environment using `requirements.txt` with, for example, pipenv.
```

Comment:
Also, we don't care for people that can't run docker, right? So put in some `development setup` instructions here instead of `running without docker`, where you just say "docker run -v $PWD/.data:/data -v $PWD:/opt/app" ...`. Also put `.data` in the `.dockerignore` and the `.gitignore` so the locally downloaded data won't appear in the image or repo.